pack/Microsoft.NET.Sdk.Functions/Microsoft.NET.Sdk.Functions.csproj

```diff
@@ -77,15 +77,15 @@
 
     <None Include="$(FunctionsGeneratorOutputPath)\netcoreapp2.0\Newtonsoft.Json.dll">
       <Pack>true</Pack>
-      <PackagePath>tools\netstandard2.0\</PackagePath>
+      <PackagePath>tools\necoreapp2.1\</PackagePath>
     </None>
     <None Include="$(FunctionsGeneratorOutputPath)\netcoreapp2.0\Microsoft.NET.Sdk.Functions.Generator.dll">
       <Pack>true</Pack>
-      <PackagePath>tools\netstandard2.0\</PackagePath>
+      <PackagePath>tools\necoreapp2.1\</PackagePath>
     </None>
     <None Include="$(FunctionsGeneratorOutputPath)\netcoreapp2.0\Microsoft.NET.Sdk.Functions.Generator.runtimeconfig.json">
       <Pack>true</Pack>
-      <PackagePath>tools\netstandard2.0\</PackagePath>
+      <PackagePath>tools\necoreapp2.1\</PackagePath>
```

Comment:
why `netcoreapp2.1` and not `netstandard2.1`? also I think there is a typo here `net`

Reply:
on never mind, that's just the path.